<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-DM7HA6N" failures="1" tests="4" name="day1jul23.DataProviderDemo" time="0.107" errors="0" timestamp="2023-07-01T10:49:44 IST" skipped="0">
  <testcase classname="day1jul23.DataProviderDemo" name="LoginApplWithDifferentUser" time="0.025"/>
  <system-out/>
  <testcase classname="day1jul23.DataProviderDemo" name="LoginApplWithDifferentUser" time="0.029">
    <failure message="Failing test for username : Abhishek" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failing test for username : Abhishek
at org.testng.Assert.fail(Assert.java:99)
at day1jul23.DataProviderDemo.LoginApplWithDifferentUser(DataProviderDemo.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- LoginApplWithDifferentUser -->
  <system-out/>
  <testcase classname="day1jul23.DataProviderDemo" name="LoginApplWithDifferentUser" time="0.026"/>
  <system-out/>
  <testcase classname="day1jul23.DataProviderDemo" name="LoginApplWithDifferentUser" time="0.027"/>
  <system-out/>
</testsuite> <!-- day1jul23.DataProviderDemo -->
